### ISSUE TITLE
Pragma cannot be set when runtime is automatic (Nextjs)

### ISSUE THREAD

**Issue by danieldelcore** [atlassian/pragmatic-drag-and-drop#85]
<img width="873" alt="image" src="https://github.com/atlassian/pragmatic-drag-and-drop/assets/3030010/0b20d51f-9d39-4afa-8ae2-65254599e0df">

```
Error: 
  × pragma cannot be set when runtime is automatic
   ╭─[/Users/foo/Development/bar/node_modules/@atlaskit/pragmatic-drag-and-drop-react-accessibility/dist/esm/drag-handle-button-base.js:1:1]
 1 │ import _extends from "@babel/runtime/helpers/extends";
 2 │ import _objectWithoutProperties from "@babel/runtime/helpers/objectWithoutProperties";
 3 │ var _excluded = ["children", "isSelected", "testId", "appearance", "type"];
 4 │ /** @jsx jsx */
   · ───────────────
 5 │ 
 6 │ import { forwardRef } from 'react';
   ╰────
```

Env: 
- React: 18
- Nextjs: 14

**Comment by benjosua** [comment]
I would try to help but im not able to reproduce this, could you provide a link to a public GitHub repository or a CodeSandbox with a minimal reproduction?

**Comment by username14415** [comment]
duplicate
https://github.com/atlassian/pragmatic-drag-and-drop/issues/67
atlassian hasn't fixed it for years, this topic is from 2022: https://community.developer.atlassian.com/t/unable-to-consume-atlaskit-in-react-18/58519

**Comment by alexreardon** [comment]
@danieldelcore could this be something to do with emotion? I think that component is using emotion 🤔

**Comment by danieldelcore** [comment]
Yeah emotion was my first guess, I'm using a newer version of react via nextjs, so I'm probably running into emotion's compatibility issues with newer react versions. 🤔

**Comment by alexreardon** [comment]
Maybe your quickest option is to copy base the drag handle button and use your own styling approach

**Comment by alexreardon** [comment]
Let me know how you go

**Comment by lucarampi** [comment]
As far as I researched, it's an Emotion-related issue. I had the same problem and I'm not entirely sure how I solved it, but my guess is that adding "use client" might have done the trick.

You can take a look [here](https://github.com/lucarampi/pragmatic-dnd-issues) if it helps.
